ArjunaCore/arjuna/tests/byteman-scripts/RecoverySuspendTest/recoverySuspendTest_FailTest.btm

```diff
@@ -38,9 +38,9 @@ ENDRULE
 RULE failIfTooManyRecoveryCycles
 CLASS com.arjuna.ats.internal.arjuna.recovery.PeriodicRecovery
 METHOD suspendScan
-AT EXIT
+AT READ _workLeftToDo
```

Comment:
I am going to approve this but out of interest does it just cause the test to fail a bit quicker?

Reply:
The test won't fail if the script waits until the end of the `suspendScan` method

Reply:
Thank you